pkg/api/v1/schema.go

```diff
@@ -803,6 +803,7 @@ type Interface struct {
 type InterfaceBindingMethod struct {
 	Bridge *InterfaceBridge `json:"bridge,omitempty"`
 	Slirp  *InterfaceSlirp  `json:"slirp,omitempty"`
+	Sriov  *InterfaceSriov  `json:"sriov,omitempty"`
```

Comment:
@rmohr Isn't Slirp also an abbreviation?

Reply:
Maybe but at least on the main page it is written as `Slirp`: http://slirp.sourceforge.net/

Reply:
I assume, SRIOV means a SRIOV VF? Maybe we should specify that we are asking for a VF.

Reply:
I can use SRIOV, no problem with that; it just looks a bit odd in the context, but ok.

@vladikr AFAIU the DP also allows to configure CRD so that it allocates the whole PF. (I haven't tested it but there is pfOnly attribute in CNI config map.)